src/index.tsx

```diff
@@ -59,9 +68,9 @@ export * from './components/molecules/inputs/Input/Input'
 export * from './components/organisms/Form/Form'
 export * from './components/organisms/Menu/Menu'
 export * from './components/organisms/ProductItem/ProductItem'
-export * from './components/organisms/Table/Table'
-export * from './components/organisms/Table/SortableTable'
-export * from './components/organisms/Table/DraggableTable'
+// export * from './components/organisms/Table/Table'
```

Comment:
commented code should be removed, or add a hint that this will later be added again